app/src/main/java/org/mozilla/focus/menu/browser/BrowserMenuAdapter.kt

```diff
@@ -69,48 +69,48 @@ class BrowserMenuAdapter(
         items.add(MenuItem.BlockingSwitch)
 
         if (customTabConfig == null || customTabConfig.showShareMenuItem) {
-            items.add(MenuItem.Default(R.id.share, resources.getString(R.string.menu_share)))
+            items.add(MenuItem.Default(R.id.share, resources.getString(R.string.menu_share), R.drawable.ic_share_menu_item))
         }
 
-        items.add(MenuItem.Default(R.id.add_to_homescreen, resources.getString(R.string.menu_add_to_home_screen)))
-        items.add(MenuItem.Default(R.id.find_in_page, resources.getString(R.string.find_in_page)))
+        items.add(MenuItem.Default(R.id.add_to_homescreen, resources.getString(R.string.menu_add_to_home_screen), R.drawable.ic_add_to_home_menu_item))
+        items.add(MenuItem.Default(R.id.find_in_page, resources.getString(R.string.find_in_page), 0))
 
         if (browsers.hasMultipleThirdPartyBrowsers(context)) {
             items.add(MenuItem.Default(R.id.open_select_browser, resources.getString(
-                    R.string.menu_open_with_a_browser2)))
+                    R.string.menu_open_with_a_browser2), R.drawable.ic_open_in_menu_item))
         }
 
         if (customTabConfig != null) {
             // "Open in Firefox Focus" to switch from a custom tab to the full-featured browser
             val appName = resources.getString(R.string.app_name)
             val label = resources.getString(R.string.menu_open_with_default_browser2, appName)
-            val menuItem = MenuItem.Default(R.id.open_in_firefox_focus, label)
+            val menuItem = MenuItem.Default(R.id.open_in_firefox_focus, label, 0)
 
             items.add(menuItem)
         }
 
         if (browsers.hasThirdPartyDefaultBrowser(context)) {
             items.add(MenuItem.Default(R.id.open_default, resources.getString(
                     R.string.menu_open_with_default_browser2, browsers.defaultBrowser!!.loadLabel(
-                    context.packageManager))))
+                    context.packageManager)), 0))
         }
 
+        items.add(MenuItem.RequestDesktopCheck)
+
         if (customTabConfig == null) {
             // There’s no need for Settings in a custom tab.
             // The user can go to the browser app itself in order to do this.
-            items.add(MenuItem.Default(R.id.settings, resources.getString(R.string.menu_settings)))
+            items.add(MenuItem.Default(R.id.settings, resources.getString(R.string.menu_settings), R.drawable.ic_settings_menu_item))
```

Comment:
\[detekt\] INFO: \[style/MaxLineLength\] Line detected that is longer than the defined maximum line length in the code style\.